Changelog 25 (#6785)
highlight/highlight: docs-content/general/changelog/changelog-25.md

---
title: Changelog 25 (10/03)
slug: changelog-25
---

> ## GitHub Settings

Reviewer: ```suggestion
## GitHub-enhanced Stacktraces
```

> ## GitHub Settings

Reviewer: ```suggestion
## GitHub Enhancement Settings
```

> We recently released GitHub-enhanced stacktraces, and now we've surfaced your GitHub settings right next to your stacktraces. It's a quality-of-life enhancement that unearths your buried settings and puts them right where you need them.

Reviewer: ```suggestion
We recently released GitHub-enhanced stacktraces, which enhances your stacktraces based on the actual content in your github account. It's a quality-of-life enhancement that unearths the actual code from an error.
```

> We recently released GitHub-enhanced stack traces, and now we've surfaced your GitHub settings right next to your stack traces. It's a quality-of-life enhancement that unearths your buried settings and puts them right where you need them.

Reviewer: ```suggestion
We recently released GitHub-enhanced stack traces, and now we've surfaced your enhancement settings right next to your stack traces. It's a quality-of-life enhancement that unearths your buried settings and puts them right where you need them.
```
